### DISCUSSION TITLE
Labeling problem

### DISCUSSION THREAD

**Discussion by myl-Giser** [iTwin/itwinjs-core#6203] [Q&A] [answered]
How to modify the label content that pops up when the mouse is placed on the element？
![image](https://github.com/iTwin/itwinjs-core/assets/81339378/eec06357-508a-4c32-99c7-59dfde02842f)

**Comment by pmconne** [comment, marked as answer]
You can [register a tooltip provider](https://www.itwinjs.org/reference/core-frontend/views/viewmanager/addtooltipprovider/).
